models/lung_nodule_ct_detection/configs/inference.json

```diff
@@ -0,0 +1,200 @@
+                10,
+                10,
+                6
+            ]
+        ]
+    },
+    "backbone": "$monai.networks.nets.resnet.resnet50(spatial_dims=3,n_input_channels=1,conv1_t_stride=[2,2,1],conv1_t_size=[7,7,7])",
+    "feature_extractor": "$monai.apps.detection.networks.retinanet_network.resnet_fpn_feature_extractor(@backbone,3,False,[1,2],None)",
+    "network_def": {
+        "_target_": "RetinaNet",
+        "spatial_dims": 3,
+        "num_classes": 1,
+        "num_anchors": 3,
+        "feature_extractor": "@feature_extractor",
+        "size_divisible": [
+            16,
+            16,
+            8
+        ]
+    },
+    "network": "$@network_def.to(@device)",
+    "detector": {
+        "_target_": "RetinaNetDetector",
+        "network": "@network",
+        "anchor_generator": "@anchor_generator",
+        "debug": false
+    },
+    "detector_ops": [
+        "$@detector.set_target_keys(box_key='box', label_key='label')",
+        "$@detector.set_box_selector_parameters(score_thresh=0.02,topk_candidates_per_level=1000,nms_thresh=0.22,detections_per_img=300)",
+        "$@detector.set_sliding_window_inferer(roi_size=@val_patch_size,overlap=0.25,sw_batch_size=1,mode='constant',device='cpu')"
+    ],
+    "preprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "DeleteItemsd",
+                "keys": [
+                    "box",
+                    "label"
+                ]
+            },
+            {
+                "_target_": "LoadImaged",
+                "keys": "image",
+                "meta_key_postfix": "meta_dict"
+            },
+            {
+                "_target_": "EnsureChannelFirstd",
+                "keys": "image",
+                "meta_key_postfix": "meta_dict"
+            },
+            {
+                "_target_": "Orientationd",
+                "keys": "image",
+                "axcodes": "RAS"
+            },
+            {
+                "_target_": "ScaleIntensityRanged",
+                "keys": "image",
+                "a_min": -1024.0,
+                "a_max": 300.0,
+                "b_min": 0.0,
+                "b_max": 1.0,
+                "clip": true
+            },
+            {
+                "_target_": "EnsureTyped",
+                "keys": "image"
+            }
+        ]
+    },
+    "dataset": {
+        "_target_": "Dataset",
+        "data": "$@test_datalist",
+        "transform": "@preprocessing"
+    },
+    "dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@dataset",
+        "batch_size": 1,
+        "shuffle": false,
+        "num_workers": 4,
+        "collate_fn": "$monai.data.utils.no_collation"
+    },
+    "inferer": {
+        "_target_": "SlidingWindowInferer",
+        "roi_size": [
+            240,
+            240,
+            160
+        ],
+        "sw_batch_size": 1,
+        "overlap": 0.5
+    },
+    "output_transform": {
```

Comment:
Should the attribute name be the more common/agreed on keyword, `postprocessing`, like `preprocessing`? In any case, I inspected this bundle and used the built-in name mapping mechanism to work around this.

Reply:
Thanks! `SlidingWindowInferer` is a monai function, the attribute name is not editable in bundle